bindata/network/ovn-kubernetes/managed/ovnkube-node.yaml

```diff
@@ -456,6 +462,7 @@ spec:
             --export-ovs-metrics \
             --disable-snat-multiple-gws \
             ${export_network_flows_flags} \
+            ${multi_network_enabled_flag} \
```

Comment:
This flag is redundant right? You enable multi-network in the configmap anyway

Reply:
I honestly don't know ... 

I thought I had to update the command. Are you saying that just by `echo`ing the string above it'll cause a rollout ? 

That explains the PR you showed me !!!

Reply:
yup, if the spec changes then a rollout will happen :)

Reply:
Thank you for leading me to the correct solution; should have understood it the first time.